packages/itwinui-css/src/file-upload/file-upload.scss

```diff
@@ -69,8 +70,120 @@
     visibility: visible;
 
     > svg,
-    .iui-template-icon {
+    .iui-template-icon,
+    .iui-file-card-icon,
+    .iui-file-card-empty-icon {
       fill: var(--iui-color-icon-accent);
     }
   }
 }
+
+@mixin iui-file-card {
+  @include iui-reset;
+  width: 100%;
+  line-height: normal;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: calc(var(--iui-size-s) * 0.5) var(--iui-size-s);
+  border-radius: var(--iui-border-radius-1);
+
+  &:focus-within {
+    outline: 2px solid var(--iui-color-text-accent);
+    outline-offset: -2px;
+  }
+
+  > svg {
+    @include iui-icon-style('xl');
+    @include iui-icon-color;
+    margin-top: var(--iui-size-s);
+    margin-bottom: var(--iui-size-s);
+    margin-left: var(--iui-size-xs);
+  }
+}
+
+@mixin iui-file-card-icon {
+  @include iui-icon-style('xl');
+  @include iui-icon-color;
+  margin-top: var(--iui-size-s);
+  margin-bottom: var(--iui-size-s);
+  margin-left: var(--iui-size-xs);
+}
+
+@mixin iui-file-card-text {
+  margin: 0 var(--iui-size-s);
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+}
+
+@mixin iui-file-card-title {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+}
+
+@mixin iui-file-card-description {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  font-size: var(--iui-font-size-0);
+  color: var(--iui-color-text-muted);
+}
+
+@mixin iui-file-card-action {
+  margin-left: auto;
+  margin-right: var(--iui-size-xs);
+  font-weight: var(--iui-font-weight-bold);
+}
+
+@mixin iui-file-card-input {
+  @include visually-hidden;
```

Comment:
you can use the `iui-visually-hidden` class directly. no need for creating a new class/mixin

Reply:
Good catch! This has been changed

Reply:
that didn't really change anything. i'm saying you don't need to make a `iui-file-card-input` because you can use `iui-visually-hidden` directly in the markup

Reply:
Even better. This has been changed